lib/game_team.rb

```diff
@@ -1,4 +1,5 @@
 class GameTeam
+   attr_reader :game_id, :team_id, :hoa, :result, :head_coach, :goals, :shots, :tackles
    def initialize(game_team_details)
       @game_id = game_team_details[:game_id]
```

Comment:
I found it helpful for future testing to change strings that return as a number (i.e. game_id => "2012030221") to change returns to integers earlier in the process. Personally I would change the return values here in the initialize.